holidays/countries/cocos_islands.py

```
@@ -0,0 +1,262 @@
+        2016: (JUL, 6),
+        2017: (JUN, 24),
+        2019: (JUN, 5),
+        2020: (MAY, 24),
+        2021: (MAY, 13),
+        2022: (MAY, 3),
+        2023: (APR, 22),
+        2024: (APR, 10),
+        2025: (MAR, 31),
+        2026: (MAR, 20),
+    }
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)